### PR TITLE
Remember relay password on receive too #177

### PR DESCRIPTION
**Pull request by maximbaz**

### CODE REVIEW

File: src/cli/cli.go [hunk 1]
```diff
@@ -334,6 +334,9 @@ func receive(c *cli.Context) (err error) {
 		if crocOptions.SharedSecret == "" {
 			crocOptions.SharedSecret = rememberedOptions.SharedSecret
 		}
+		if !c.IsSet("pass") {
```
Comment (maximbaz) [inline src/cli/cli.go]: I'm only not sure what's the difference to `GlobalIsSet`, I saw you used a regular `IsSet` for the `send` command, that's why I did the same

Comment (schollz) [reply src/cli/cli.go]: Thanks @maximbaz. I made a mistake! Both of them should be `GlobalIsSet`. You can update your PR or I can just do it :)

Comment (maximbaz) [reply src/cli/cli.go]: Updated 👍

**Comment by schollz** [comment]
Thanks! Really appreciate that you caught my mistakes :)

**Pull request merged**